src/inference/src/dev/core_impl.cpp

```diff
@@ -994,6 +994,18 @@ ov::SoPtr<ov::IRemoteContext> ov::CoreImpl::get_default_context(const std::strin
     return get_plugin(parsed._deviceName).get_default_context(parsed._config);
 }
 
+void validate_batch_device_name(const std::string& deviceName) {
+    auto name = deviceName;
+    auto pos = name.find(",");
+    // for trailing comma such as "gpu,", we do not treat it as multi-device.
```

Comment:
For similar "gpu," case, we should treat it as invalid device to throw exception rather than fix it.

Reply:
I've modified the logic here, thank you for the comment!